README.md

```diff
@@ -1007,6 +1007,46 @@ rather than making external requests.
 response = client.get('http://www.example.com/hello_world/')
 ```
 
+The TestClient also have a post method. There are two ways to send content to
+the post method. Using the data argument or using the json argument.
+
+Using the data argument:
+
+```python
+from app import app
+from apistar import TestClient
+
+def test_hello_world():
+	response = client.post('/hello_world', data="content")
+    assert response.json() == {'hello_world': 'content'}
+```
+
+It is possible to send a dict as the data argument:
+
+```python
+from app import app
+from apistar import TestClient
+
+def test_hello_world():
+	response = client.post('/hello_world', data={"title": "My Title"})
+    assert response.json() == {'hello_world': {'title': 'My Title'}}
+```
+
+The json argument is similar:
```

Comment:
You might also document the difference between data and json and when to use which.